protocol/p2p/src/core/router.rs

```diff
@@ -231,3 +232,13 @@ impl Router {
         true
     }
 }
+
+fn msg_type_string(msg: &KaspadMessage) -> String {
```

Comment:
I prefer avoiding the String heap alloc. Maybe return `Option<KaspadMessagePayloadType>` and rely on a debug print above?